webapp/channels/NOTICE.txt

```diff
@@ -3653,42 +3670,6 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
-
----
-
-## sass
```

Comment:
We're also still using `sass` as well

Reply:
It did move from the `dependencies` in `webapp/channels/package.json` to the `devDependencies` in `webapp/package.json` which might've caused it to get missed from here. I forget what our policy on including dev tools in this file is since it should've probably always been listed as a dev dependency